Lib/test/test_type_aliases.py

```diff
@@ -211,6 +211,19 @@ def test_generic(self):
         self.assertEqual(TA.__value__, list[T])
         self.assertEqual(TA.__type_params__, (T,))
         self.assertEqual(TA.__module__, __name__)
+        self.assertIs(TA[int].__class__, types.GenericAlias)
```

Comment:
```suggestion
        self.assertIs(type(TA[int]), types.GenericAlias)
```
The `__class__` attribute may be a lie, let's check the real type.